sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -201,6 +201,14 @@ object SQLConf {
     .stringConf
     .createOptional
 
+  val OPTIMIZER_PLAN_CHANGE_LOG_BATCHES = buildConf("spark.sql.optimizer.planChangeLog.batches")
+    .internal()
+    .doc("If this configuration is set, the optimizer will only log plan changes caused by " +
+      "applying the batch specified in this configuration. The value can be a list of batch " +
+      "names separated by comma.")
```

Comment:
How about this?
```
    .doc("Configures a list of batches to be logged in the optimizer, in which the batches are " +
      "specified by their batch names and separated by comma.")
```

Reply:
can I update similar description for "spark.sql.optimizer.planChangeLog.rules" config? @maryannxue  any suggestion?